app/src/main/java/ingsw/group1/findmyphone/event/SMSLogEvent.java

```diff
@@ -3,6 +3,8 @@
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 
+import com.eis.smslibrary.SMSPeer;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'com.eis.smslibrary.SMSPeer'](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=5052095)

Reply:
There are these useful checks on android studio, so that you can avoid these things
![image](https://user-images.githubusercontent.com/56435959/75160777-f0445e00-571a-11ea-9a99-c2cdb0e5989a.png)

Reply:
Thank you, i didn't know about that